src/docs/schema.dox

```diff
@@ -362,4 +362,16 @@ Here is another example program, @ex_ref{ex_call_center.c}.
 @snippet ex_call_center.c call-center decl
 @snippet ex_call_center.c call-center work
 
+@section logged_non_logged_tables Logged and Non-logged tables
+
+- WiredTiger writes records to the log for each transaction when logging is enabled
```

Comment:
How about something like the following?
`WiredTiger writes records to the log for each transaction commit when logging is enabled on the tables`

Reply:
Fixed